src/braket/default_simulator/openqasm/interpreter.py

```diff
@@ -470,11 +470,16 @@ def _(self, node: QuantumGateModifier) -> QuantumGateModifier:
 
     @visit.register
     def _(self, node: QuantumMeasurement) -> None:
-        """Doesn't do anything, but may add more functionality in the future"""
+        qubit = node.qubit
+        if isinstance(qubit, IndexedIdentifier):
+            qubit_value = qubit.indices[0][0].value
+            self.context.add_measure(qubit_value)
+        else:  # Identifier
+            self.context.add_measure()
```

Comment:
looks like this case is missing an arg. I believe both cases can be simplified to just one using the helper `self.context.get_qubits(node.qubit)`, which takes an identifier (possibly indexed)

Reply:
^`get_qubits` will return a tuple of qubit indices, and can be used even with physical qubits (`$5`)

Reply:
That is a great point, there is no reason to have add_measure not take an arg. I'll update add_measure to take in a required tuple and use `self.context.get_qubits(node.qubit)` here.